docker_mcp/core/transfer/archive.py

```
@@ -56,6 +58,74 @@
     
     def __init__(self):
         self.logger = logger.bind(component="archive_utils")
+        self.safety = MigrationSafety()
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)